docs/Toy/toy/parser.py

```diff
@@ -71,7 +71,8 @@ def getTokenPrecedence(self) -> int:
             return -1
 
     def peek(self, pattern: str | type[Token] | None = None) -> None:
-        """Verifies that the current token fits the pattern, raises ParseError otherwise"""
+        """Verifies that the current token fits the pattern, "
```

Comment:
I think those will introduce `"` in the middle of the documentation, since the delimiters are `"""`

Reply:
Right, thanks